### ISSUE TITLE
Move table header definitions to table header

### ISSUE THREAD

**Issue by skliper** [nasa/SC#16]
SC_RtsEntryHeader_t and SC_AtsEntryHeader_t are currently defined in sc_app.h:
https://aetd-git.gsfc.nasa.gov/gsfc-cfs/cfs_sc/-/blob/develop/fsw/src/sc_app.h#L306
https://aetd-git.gsfc.nasa.gov/gsfc-cfs/cfs_sc/-/blob/develop/fsw/src/sc_app.h#L330

Really should be owned by sc_tbldefs.h so that tables that use them don't need to include the entire sc_app.h.

Imported from GSFCCFS-1840

**Comment by jphickey** [comment]
Just discovered this one, its a real problem.  The tables are defined as an array of `uint16` values, but read by the FSW as an instance of `SC_RtsEntryHeader_t` as is done here:

https://github.com/nasa/SC/blob/b59d70a6b42acc9bcaac47ba38ddc0bd8913cc7a/fsw/src/sc_rtsrq.c#L104

But the "TimeTag" value here is a `uint32`, not a `uint16`: https://github.com/nasa/SC/blob/b59d70a6b42acc9bcaac47ba38ddc0bd8913cc7a/fsw/src/sc_tbldefs.h#L59

Therefore, on a little endian machine, if the TimeTag is not 0, the value is incorrect.  The default RTS examples (e.g. rts001, rts002) both use a value of 0 for the TimeTag, so the bug gets hidden - it never attempts to send the other 2 commands in the RTS, that have nonzero values here.  

I changed CMD1_TIME from 0 to 1, and sure enough, the `SC_ComputeAbsTime` is called with a value of 0x1000 (65536), rather than 1, because it was incorrectly cast.

It looks like ATS was updated to split the time tag into a 16-bit MS and LS word (a hack, but will at least get the logical value to come through), but RTS was not updated in the same manner.
